http/src/main/java/com/hazelcast/jet/contrib/http/HttpSinks.java

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright (c) 2008-2020, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.jet.contrib.http;
+
+import com.hazelcast.jet.JetInstance;
+import com.hazelcast.jet.pipeline.Sink;
+import com.hazelcast.partition.Partition;
+import com.hazelcast.partition.PartitionService;
+
+import javax.annotation.Nonnull;
+
+/**
+ * Contains factory methods for creating WebSocket and Server-Sent Events server sinks.
+ * Clients can connect to the sinks and stream the results of the pipeline.
+ * <p>
+ * Server addresses can be retrieved from {@link #webSocketAddress(JetInstance)} and
+ * {@link #sseAddress(JetInstance)} (JetInstance, Job)} respectively.
+ */
+public final class HttpSinks {
```

Comment:
When I saw HttpSinks I expected an HTTP request being made for each item to a 3rd party service, the websocket endpoint is a different direction.

Reply:
I'll rename current sinks as `HttpListenerSinks` and add the ones you've mentioned as `HttpSinks`, as a separate PR.